### DISCUSSION TITLE
Error: Failed to send transaction || error code -32000: insufficient funds for gas * price + value: balance 7

### DISCUSSION THREAD

**Discussion by vigneshbalu96** [Cyfrin/foundry-full-course-cu#2961] [Q&A] [answered]
Getting error while running the `forge script' command...

![Screenshot from 2024-12-03 05-55-36](https://github.com/user-attachments/assets/2405bd66-5718-4ae0-9793-b448f2d32a3c)
![Screenshot from 2024-12-03 05-55-57](https://github.com/user-attachments/assets/40a52f06-e7b8-41e7-9f5c-6e1baea2016f)
![Screenshot from 2024-12-03 05-56-49](https://github.com/user-attachments/assets/f14dfeb9-7c50-4acf-ae48-30771efa1af0)

Thanks in Advance! :smile:

**Comment by vigneshbalu96** [comment]
Do we need to encrypt the private_key of the Anvil chain or Sepolia chain with the command `cast wallet import default --interaction` to run the command `forge script script/Interactions.s.sol:FundSubscription --rpc-url $SEPOLIA_RPC_URL --account default --broadcast`?

**Reply by EngrPips** [reply]
Yes, You must have initially encrypted the private key before you can request it from Foundry.

**Comment by heheboii11** [comment]
Ig you are not having enough funds in your wallet to send the transaction, please get the sepolia eth from the faucet and try again

**Comment by vigneshbalu96** [comment, marked as answer]
The issue with the error was I used the _**anvil chain**_ account to execute the `forge script` command, with the **_sepolia account_** it works as expected!

**Reply by EngrPips** [reply]
Awesome, I am glad you figured it out.
